### ISSUE TITLE
[VarDumper] Can't find variable: Sfdump JS Error

### ISSUE THREAD

**Issue by PhilETaylor** [symfony/symfony#48039]
### Symfony version(s) affected

5.4+

### Description

On the Logs page of the Symfony Profiler when an exception has been thrown, the dump of traces creates a javascript console error

### How to reproduce

So.... To replicate create a demo app with Symfony 5.4

```
composer create-project symfony/symfony-demo mySymfony54 1.8.0
cd mySymfony54
symfony serve
# Visit http://127.0.0.1:8000/en/blog/ to confirm its working
# Edit BlogController.php and throw an \Exception('test') in the index method
# Visit http://127.0.0.1:8000/en/blog/ to confirm you see the exception 
# Click the toolbar at the bottom to get to the Symfony Profiler
# Click LOGS in the left menu 
# Check console.log 
```

see console.log: 

<img width="305" alt="Screenshot 2022-10-29 at 00 14 52" src="https://user-images.githubusercontent.com/400092/198749293-47ae5000-d210-4577-85c7-664045766225.png">

Click to open the error message and see overlapping text and no pretty-ness.

<img width="1247" alt="Screenshot 2022-10-29 at 00 15 10" src="https://user-images.githubusercontent.com/400092/198749346-917904f7-c4d1-4195-8e0c-0fbdc33fc99e.png">

Once this PR is applied you then can repeat and you will see no console.log error message and you will see a pretty output like:

<img width="1096" alt="Screenshot 2022-10-29 at 00 16 24" src="https://user-images.githubusercontent.com/400092/198749391-04d722ad-2638-43bb-9cf9-aef3e1dece9a.png">

### Possible Solution

I attempted https://github.com/symfony/symfony/pull/48038 but that was wrong and reverted good work by @nicolas-grekas years ago - sorry, I failed.

**Comment by PhilETaylor** [comment]
(This seems fixed in 6.2, but still a problem in 5.4 and 6.1 series)

**Comment by PhilETaylor** [comment]
fixed by #48048
